src/metadata.rs

```diff
@@ -26,6 +26,7 @@ use crate::providers::ibmcloud::IBMGen2Provider;
 use crate::providers::ibmcloud_classic::IBMClassicProvider;
 use crate::providers::openstack::network::OpenstackProvider;
 use crate::providers::packet::PacketProvider;
+#[cfg(feature = "cl-legacy")]
```

Comment:
Should the `pub mod` in `src/providers/mod.rs` also be flagged?

Reply:
Absolutely yes, and clippy confirmed it too.